File: psets/pset6/mario/more/mario.py
from cs50 import get_int

def main():
    height = get_int("Height: ")

    while(height > 8 or height < 1):
        height = get_int("Height: ")

    for i in range(height):
        for j in range(height - i - 1):
            print(" ", end="")

        for j in range(height - ((height - 1) - i)):
            print("#", end="")

        print()

main()

File: psets/pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do {
       height = get_int("Height: ");
    }
    while (height > 8 || height < 1);
    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < height - (i + 1); j++)
        {
            printf(" ");
        }

        for (int j = 0; j < height - ((height - 1) - i); j++)
        {
            printf("#");
        }

      //  printf("  ");

        for (int j = 0; j < height - ((height - 1) - i); j++)
        {
            printf("#");
        }
        printf("\n");
    }
}